lib/ffmpegCommandService.js

```diff
@@ -112,6 +112,11 @@ class FfmpegCommandService {
             fadeOut: whippleCommandFadeEffectOptional
         });
 
+        const whippleCommandTextAlignment = Joi.object().keys({
```

Comment:
What's the use case for using text alignment vs x/y location?

Reply:
Basically it isn't possible to center right/bottom align without knowing the bounding box of the text. `ffmpeg` is the only process that has knowledge of this. So instead of saying e.g. `y=600` we use special `ffmpeg` syntax such as `y=(main_h/2-text_h/2)`

Reply:
Ah, alrighty.  Would it make sense to add these options directly to xLoc/yLoc?

Reply:
Looks like that's how it's getting used down below in the code.  I'd update xLoc/yLoc to allow these explicit alignment options.